src/scss/global.scss

```diff
@@ -759,3 +759,28 @@ span.link {
 .flex-one {
   flex: 1;
 }
+
+// font size
+.font-size--small {
+  font-size: 0.9rem;
+}
+
+// opacity
+.opacity-8 {
+  opacity: 0.8;
+}
+.opacity-6 {
+  opacity: 0.6;
+}
+.opacity-3 {
+  opacity: 0.3;
+}
```

Comment:
BEM:

```suggestion
.opacity {
  &--8 {
    opacity: 0.8;
  }
  
  &--6 {
    opacity: 0.6;
  }
  
  &--3 {
    opacity: 0.3;
  }
}
```

Reply:
Depending on how we end up using these though.. it's likely going to end up pushing more 'styling concerns' into the markup/JSX, which isn't something I am a particular fan of.

This might be better as an SCSS mixin/similar so that we can keep our styles to our stylesheets, and out of our JSX:

- https://sass-lang.com/documentation/at-rules/mixin

Reply:
agree here, removed `.opacity` class names. https://github.com/sparkletown/sparkle/pull/1328/commits/e21e67445665543be0fab0f660743356141c30c7